From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sun, 17 Jun 2018 21:17:34 +0200
Subject: [PATCH] {MAYBE UPS} net/bpfilter: fix unfound include

gcc reports:

    net/bpfilter/main.c:9:10: fatal error: include/uapi/linux/bpf.h: No
    such file or directory
     #include "include/uapi/linux/bpf.h"
              ^~~~~~~~~~~~~~~~~~~~~~~~~~
---
 net/bpfilter/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/bpfilter/main.c b/net/bpfilter/main.c
index 1317f108df8a..61ce8454a88e 100644
--- a/net/bpfilter/main.c
+++ b/net/bpfilter/main.c
@@ -6,7 +6,7 @@
 #include <sys/socket.h>
 #include <fcntl.h>
 #include <unistd.h>
-#include "include/uapi/linux/bpf.h"
+#include "../../include/uapi/linux/bpf.h"
 #include <asm/unistd.h>
 #include "msgfmt.h"
 
-- 
